src/main/java/com/minecolonies/coremod/entity/ai/minimal/EntityAISleep.java

```diff
@@ -60,7 +60,7 @@ public boolean shouldExecute()
      * @return true while he should sleep.
      */
     @Override
-    public boolean continueExecuting()
+    public boolean shouldContinueExecuting()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 25 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)